packages/vitest/src/node/core.ts

```diff
@@ -61,9 +61,10 @@ export class Vitest {
     this.state = new StateManager()
     this.snapshot = new SnapshotManager(resolved)
     // @ts-expect-error cli type
-    this.reporters = toArray(resolved.reporters || resolved.reporter)
+    this.reporters = toArray(resolved.reporter || resolved.reporters)
```

Comment:
Shouldn't we directly use `resolved.reporters` here and overwrite it with `reporter` inside `resolveConfig`?

Reply:
There are (I think 3 possibles solution)

- current propossal
- your propossal changes (makes sense to me too)
- rename cli option from reporter to reporters (because with this change  https://github.com/vitest-dev/vitest/pull/720 reporters could be now an array of reporters (make sense that?)) 

What is the best solution here?

Reply:
We should keep the default of the `reporters` option empty and resolve them in `resolveConfig`. When the array is empty, append `default` to it.